src/main/kotlin/org/jetbrains/research/ml/codetracker/server/CollectionsQueryExecutor.kt

```diff
@@ -19,7 +19,22 @@ object CollectionsQueryExecutor : QueryExecutor() {
     }
 
     inline fun <reified T : Any> getCollection(url: String, serializer: KSerializer<T>): List<T> {
-        return parseResponse(
+        return parseCollectionResponse(
+            executeQuery(
+                Request.Builder().url(URL("${baseUrl}${url}")).build()
+            ), serializer
+        )
```

Comment:
These lines are repeated twice (here and at lines 38-40), maybe we should move them out to some private method? 
for example,
```
    private fun getResponse(url: String): Response? {
        return executeQuery(Request.Builder().url(URL("${baseUrl}${url}")).build())
    }
```